From 8dc1e0b04b316e0e860e39a61d012fd9772bf32c Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Wed, 26 Nov 2025 18:35:24 +0100
Subject: [PATCH] Revert doc/ part of "src: add tcp option reset support"

JIRA: https://issues.redhat.com/browse/RHEL-126817
Upstream Status: RHEL-only

Revert documentation added by commit
5d837d270d5a8b3a4d3fdca12d0f0800b8287cdd.

Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 doc/statements.txt | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/doc/statements.txt b/doc/statements.txt
index 1967280..abe9e6d 100644
--- a/doc/statements.txt
+++ b/doc/statements.txt
@@ -80,13 +80,6 @@ tcp flags syn tcp option maxseg size set 1360
 tcp flags syn tcp option maxseg size set rt mtu
 ---------------
 
-You can also remove tcp options via reset keyword.
-
-.remove tcp option
----------------
-tcp flags syn reset tcp option sack-perm
----------------
-
 LOG STATEMENT
 ~~~~~~~~~~~~~
 [verse]
